src/pug/layouts/menu.pug

```diff
@@ -0,0 +1,33 @@
+mixin menu 
+    .menu 
+        .container 
+            nav.nav.nav-bar     
+                .nav__logo
+                    a.nav__link(href="") 
+                        img.logo(src="/assets/logoMenu.png" alt="")
+                .nav__menu 
+                    ul.nav__list
+                        each item in menu 
+                            li.nav__item(class = item.class) 
+                                a.nav__link=item.item
+                                if(item.class === "child")
```

Comment:
Nhưng đừng check `class === 'child'` nhé. Những thứ như này nên đặt cho nó là 1 giá trị boolean. Ví dụ: hasSubmenu: true hoặc hasChild: true
Cách đặt tên các biến boolean có thể tham khảo ở đây nhé
https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables